.github/workflows/turborepo-release-step-3.yml

```diff
@@ -5,6 +5,9 @@ on:
     inputs:
       release_branch:
         description: "Staging branch to run release from"
+      dry_run:
+        description: "Do a dry run, skipping the final publish step."
+        type: boolean
```

Comment:
Should we default this to true for more safety?

Reply:
Good question, thoughts @tknickman?

Reply:
I think we should do this separately. I am going to merge now :)